portal.go

```diff
@@ -223,6 +223,16 @@ func (br *IMBridge) NewPortal(dbPortal *database.Portal) *Portal {
 	return portal
 }
 
+// NewPortalForBackfill creates a portal for the client-side bridge to backfill
+// into.
+func (br *IMBridge) NewPortalForBackfill(guid, threadID string) (*Portal, error) {
```

Comment:
This whole helper method is a bit pointless 🤔

Reply:
I thought I had to use the private `loadDBPortal` function, hence why I created this function, but since I don't have to do that, this can be inlined.